client/src/components/HandMarkedPaperBallot.tsx

```diff
@@ -769,121 +817,199 @@ const HandMarkedPaperBallot = ({
+                        >
+                          Vote <strong>Yes</strong> or <strong>No</strong>
+                        </Trans>
+                        {locales.secondary && (
+                          <React.Fragment>
+                            {' / '}
                             <Trans
                               i18nKey="voteYesOrNo"
-                              tOptions={{ lng: locales.primary }}
+                              tOptions={{ lng: locales.secondary }}
                             >
                               Vote <strong>Yes</strong> or <strong>No</strong>
                             </Trans>
-                            {locales.secondary && (
-                              <React.Fragment>
-                                {' / '}
-                                <Trans
-                                  i18nKey="voteYesOrNo"
-                                  tOptions={{ lng: locales.secondary }}
-                                >
-                                  Vote <strong>Yes</strong> or{' '}
-                                  <strong>No</strong>
-                                </Trans>
-                              </React.Fragment>
-                            )}
-                          </p>
-                          <Text
-                            small
-                            preLine
-                            dangerouslySetInnerHTML={{
-                              __html: contest.description,
-                            }}
-                          />
-                          {localeContests && (
-                            <Text
-                              small
-                              preLine
-                              dangerouslySetInnerHTML={{
-                                __html: (localeContests[i] as YesNoContest)
-                                  .description,
-                              }}
-                            />
-                          )}
-                          {['Yes', 'No'].map((answer) => (
-                            <Text key={answer} bold noWrap>
-                              <BubbleMark
-                                checked={
-                                  votes[contest.id] &&
-                                  votes[contest.id]![0] === answer.toLowerCase()
-                                }
-                              >
-                                <span>{dualLanguageWithSlash(answer)}</span>
-                              </BubbleMark>
-                            </Text>
-                          ))}
-                        </React.Fragment>
+                          </React.Fragment>
+                        )}
+                      </p>
+                      <Text
+                        small
+                        preLine
+                        dangerouslySetInnerHTML={{
+                          __html: contest.description,
+                        }}
```

Comment:
We're not sanitizing this at all?

Reply:
Not yet!

Reply:
there's a ticket for it:
https://github.com/votingworks/vxmail/issues/143

Reply:
Adding this now.